src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -1,5 +1,11 @@
 To work well with server-rendered pages, Amplify JS requires slight modifications from how you would use it in a client-only environment.
 
+<Callout>
+
+You cannot use Amplify JS to run authenticated requests using server side functions that do not have a `req` object such as `getStaticPaths` or `getStaticProps` from NextJS.
```

Comment:
I think that sounds good, but let's move it to the `withSSRContext` section on [this line](https://github.com/aws-amplify/docs/blob/f3d7583b48641516d1f38b7c6022534faaf8f2c4/src/fragments/lib/ssr/js/getting-started.mdx?plain=1#L74). 

Combining the two sentences to avoid repetition, it can say something like (please edit as needed):

Server-side functions that _don't_ have a `req` object (e.g. Next.js' `getStaticProps` & `getStaticPaths`) should still use `withSSRContext()`. Please note that it's not possible to perform authenticated requests with Amplify when using these functions.